pubspec.yaml

```diff
@@ -8,15 +8,19 @@ documentation: https://docs.revenuecat.com/
 
 environment:
   sdk: ">=2.1.0 <3.0.0"
-  flutter: ">=1.12.0 <2.0.0"
+  flutter: ">=1.12.13+hotfix.6 <2.0.0"
```

Comment:
Also taken from the upgrading docs, this is the minimum version they offer support for.